src/app/globals.css

```diff
@@ -79,7 +79,9 @@ html {
 }
 
 .playerBackgroundElementOnTop {
-  background-image: url('https://lh3.googleusercontent.com/d/114_RLl18MAzX035svMyvNJpE3ArfLNCF=w500');
+  background-size: cover;
+  background-position: center;
+  background-image: url('https://github.com/user-attachments/assets/7627cd1b-371c-4912-92bc-db5411979731');
```

Comment:
Why did you change this image? The new one is missing a part of the tail of Cheddy.

Reply:
the older URL sometimes worked, sometimes it threw error, so I uploaded it to github server and use that link, if you have the picture, can you please share, I will upload it and use that link then.

Reply:
I don't have it. Maybe Blaze do.